paper/paper.md

```diff
@@ -70,40 +70,34 @@ for species, agents in grouped:
+Mesa 3 provides multiple spatial environments where agents interact, offering both discrete (cell-based) and continuous space implementations:
 
-Mesa comes with a wide variety of discrete spaces, including OrthogonalMooreGrid, OrthogonalVonNeumanGrid, Hexgrid, Network, and VoroinoiMesh. These are all implemented using a doubly linked data structure where each cell has connections to its neighboring cells. The different discrete spaces differ with respect to how they are "wired-up", but the API is uniform across all of them.
+In discrete spaces, an agent occupies a cell. Mesa implements discrete spaces using a doubly-linked structure where each cell maintains connections to its neighbors. The framework includes several discrete space variants with a consistent API:
 
-Mesa also offers three subclasses of the Agent class designed to be used in conjunction with these discrete spaces: FixedAgent, CellAgent, and Grid2DMovingAgent. FixedAgent is assigned to a given cell and can access this cell via self.cell. However, it cannot be moved once assigned to a given cell. A CellAgent, like a FixedAgent, has access to the cell it currently occupies. However, it can update this attribute, making it possible to move around. A Grid2DMovingAgent extends CellAgent by offering a move method with shorthand for the direction of movement.
+- Grid-based: `OrthogonalMooreGrid`, `OrthogonalVonNeumanGrid`, and `HexGrid`
+- Network-based: `Network` for graph-based topologies
+- Voronoi-based: `VoronoiMesh` for irregular tessellations
 
-1. Grid-based spaces:
+Example grid creation:
 ```python
 grid = OrthogonalVonNeumannGrid(
     (width, height), torus=False, random=model.random
 )
-
-# create a network space with a capacity of 1 agent per node
-grid = Network(networkx_graph, capacity=1, random=model.random)
 ```
 
-2. Network spaces:
-```python
-network = NetworkGrid(networkx_graph)
-network.get_neighbors(agent, include_center=False)
-```
+Mesa provides specialized agent classes for spatial interactions in the discrete spaces:
 
-The OrthogonalMooreGrid, OrthogonalVonNeumanGrid and Hexgrid come with support for numpy based layers with additional data: PropertyLayers. Cells have attribute access to their value in each of these property layers, while the entire layer can be accessed from the space itself.
+- `FixedAgent`: Remains at its assigned cell
+- `CellAgent`: Can move between cells and access its current location
+- `Grid2DMovingAgent`: Extends `CellAgent` with directional movement methods
 
+All discrete spaces support PropertyLayers - efficient numpy-based arrays for storing environmental data:
```

Comment:
I really don't like to use the term "environmental data" or in any way suggest that MESA does environmental modeling or environmental data analysis. Not sure, however, how to reformulate this while keeping it concise. Perhaps just speak of properties of the space?

Reply:
Changed it to:

> All discrete spaces support PropertyLayers - efficient numpy-based arrays for storing cell-level properties: